src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -58,16 +53,16 @@ let isPhysicalFileKind (kind: string) =
     kind.Equals(EnvDTE.Constants.vsProjectItemKindPhysicalFile, StringComparison.OrdinalIgnoreCase)
 
 let isPhysicalFileOrFolderKind kind =
-    kind <> null && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
+    isNotNull kind && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
```

Comment:
I see. I think the perf of `=` `<>` must be fixed in the compiler. This situation is unhealthy :(